docs/topics/api/overview.rst

```diff
@@ -166,35 +166,15 @@ fallback, depending on whether a translation is available in the requested
 language) will be returned.
 
 
-.. _api-overview-translations-v3:
-
-^^^^^^^^^^^^^^^^^^^^
-Default API behavior
-^^^^^^^^^^^^^^^^^^^^
-
-In API version 3 or 4 the response, if the ``lang`` parameter is passed, is a single string.
-
-.. code-block:: json
-
-    {
-        "name": "Games"
-    }
-
-This behaviour also applies to ``POST``, ``PATCH`` and ``PUT`` requests: you
-can either submit an object containing several translations, or just a string.
-If only a string is supplied, it will only be used to translate the field in
-the current language.
-
-
 .. _api-overview-translations-v5:
 
 ^^^^^^^^^^^^^^^
 v5 API behavior
```

Comment:
Should this be changed to say "Default API behavior", because this doc is now about `v5`?

Reply:
It could?  We aren't documenting any other versions in these docs though.  (so "v5" is redundant, but "default" is too, kinda)

Reply:
That makes sense, so what should we have here?

Reply:
I'll just reword the heading away

Reply:
https://github.com/mozilla/addons-server/pull/16303/commits/97c1654fb9e739eca3fd3ff3d297eaa054090a30